seacrowd/sea_datasets/thai_toxicity_tweet/thai_toxicity_tweet.py

```diff
@@ -0,0 +1,120 @@
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@inproceedings{sirihattasak2018annotation,
+  title     = {Annotation and Classification of Toxicity for Thai Twitter},
+  author    = {Sirihattasak, Sugan and Komachi, Mamoru and Ishikawa, Hiroshi},
+  year      = {2018},
+  booktitle = {Proceedings of LREC 2018 Workshop and the 2nd Workshop on Text Analytics for Cybersecurity and Online Safety (TA-COS’18)},
+  address   = {Miyazaki, Japan},
+}
+"""
+
+_LOCAL = False
+_LANGUAGES = ["tha"]
+_DATASETNAME = "thai_toxicity_tweet"
+_DESCRIPTION = """\
+The Thai Toxicity Tweet Corpus contains 3,300 tweets (506 tweets with texts missing) annotated by humans with guidelines
+including a 44-word dictionary. The author acquired 2,027 toxic and 1,273 non-toxic tweets, which were
+manually labeled by three annotators. Toxicity is defined by the author as any message conveying harmful,
+damaging, or negative intent, in accordance with their defined criteria for toxicity.
+"""
+
+_HOMEPAGE = "https://github.com/tmu-nlp/ThaiToxicityTweetCorpus"
+_LICENSE = Licenses.CC_BY_NC_4_0.value
+_URL = "https://huggingface.co/datasets/thai_toxicity_tweet"
+
+_SUPPORTED_TASKS = [Tasks.ABUSIVE_LANGUAGE_PREDICTION]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class ThaiToxicityTweetsDataset(datasets.GeneratorBasedBuilder):
+    """Dataset of Thai tweets annotated for toxicity."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+    SEACROWD_SCHEMA = "text"
```

Comment:
Since you have already defined `_SUPPORTED_TASKS`, I think you could use it for `SEACROWD_SCHEMA` instead of hardcoding it. 

See PR #293